### ISSUE TITLE
RBEL failed: The given URL is invalid.

### ISSUE THREAD

**Issue by tobiha** [gematik/app-Tiger#3]
Since upgrading `tiger-standalone-proxy`  from version `0.15.0` to `0.18.1` I've been getting the following errors 

```
ERROR 1 --- [ver-Scheduler13] d.g.t.tiger.proxy.ForwardProxyCallback   : RBel FAILED!

de.gematik.rbellogger.exceptions.RbelConversionException: The given URL is invalid. Please check your configuration.
	at de.gematik.rbellogger.data.RbelHostname.checkIfUrlIsValid(RbelHostname.java:104) ~[rbellogger-0.22.0.jar!/:na]
	at de.gematik.rbellogger.data.RbelHostname.generateFromUrl(RbelHostname.java:83) ~[rbellogger-0.22.0.jar!/:na]
	at de.gematik.test.tiger.proxy.MockServerToRbelConverter.convertUri(MockServerToRbelConverter.java:69) ~[tiger-proxy-0.18.1.jar!/:na]
	at de.gematik.test.tiger.proxy.MockServerToRbelConverter.convertRequest(MockServerToRbelConverter.java:53) ~[tiger-proxy-0.18.1.jar!/:na]
	at de.gematik.test.tiger.proxy.ForwardProxyCallback.handleResponse(ForwardProxyCallback.java:56) ~[tiger-proxy-0.18.1.jar!/:na]
	at de.gematik.test.tiger.proxy.AbstractTigerRouteCallback.handle(AbstractTigerRouteCallback.java:114) ~[tiger-proxy-0.18.1.jar!/:na]
	at org.mockserver.mock.action.http.HttpForwardObjectCallbackActionHandler.lambda$handleLocally$0(HttpForwardObjectCallbackActionHandler.java:70) ~[mockserver-core-5.11.9.jar!/:5.11.9]
	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:859) ~[na:na]
	at java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:837) ~[na:na]
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:478) ~[na:na]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[na:na]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:829) ~[na:na]
```

I haven't changed any configurations and it was working perfectly prior to the upgrade.

The configuration it self is very basic and I could not find any hints in the release notes regarding needed migrations:

```
server.port: 8001
tiger-proxy:
  port: 8000
  proxyRoutes:
    - id: route1
      from: "http://route.to.service:8000"
      to: "http://forwardserivce:8000"
      disableRbelLogging: false
  activateRbelEndpoint: true
  keyFolders:
    - "/default-keyfiles/"
```

**Comment by Arkinator** [comment]
The problem is a more restrictive URL-Validation which was built into Rbel recently. The used URLs have to pass https://commons.apache.org/proper/commons-validator/apidocs/org/apache/commons/validator/routines/UrlValidator.html. 

Since this seems to not work correctly on intranet or fantasy-urls (we have seen errors with custom domains) we will get rid of this validation in an upcoming release (hopefully in time for 0.19.0). This issue has been mirrored as RBEL-41 internally.

As a workaround you could alter your domain-names so they pass the commons-test.

**Comment by tobiha** [comment]
Thank you for your quick reply. Indeed we are using domains for routing which do not pass this test, but as we are running our services inside a k8s cluster, we can't change this easily. We will be waiting for the 0.19.0 release then...

**Comment by Arkinator** [comment]
fixed, will be shipped with 0.19.0
